reloj_checador_team03/app/views/test/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<form>
+<h3>Formulario ficticio</h3>
+  <div class="form-group">
+    <label for="exampleFormControlInput1">Email address</label>
+    <input type="email" class="form-control" id="exampleFormControlInput1" placeholder="name@example.com">
+  </div>
+  <div class="form-group">
+    <label for="exampleFormControlSelect1">Example select</label>
+    <select class="form-control" id="exampleFormControlSelect1">
+      <option>1</option>
```

Comment:
maybe you can use and .each method with ruby here to create the options, i don't know if it works but can be a interesting experiment